src/utils/update.rs

```diff
@@ -175,10 +184,12 @@ pub fn get_latest_sentrycli_release() -> Result<SentryCliUpdateInfo> {
     })
 }
 
+#[cfg(not(feature = "managed"))]
 pub fn can_update_sentrycli() -> bool {
```

Comment:
I wonder whether this function is even necessary. Ideally, we would distribute to package managers the `managed` build, so that this function would always return true.